### DISCUSSION TITLE
disable hot reload

### DISCUSSION THREAD

**Discussion by naftali100** [JanDeDobbeleer/oh-my-posh#1950] [General]
i noticed that when i edit the config file next time the theme rendered the changes take effect even without reloading the shell.
this mean that the file is being read every time... 
it seem a bit redundant...
Is it intentional?
is there a way to disable this reload?

**Comment by JanDeDobbeleer** [comment]
That is intentional yes. It needs to get stored anyways regardless as oh-my-posh starts, renders the prompt and stops, meaning it always requires a read. There's no real measurable overhead involved, trying to fix this will not necessarily result in a performance gain..

**Reply by naftali100** [reply]
oh... ok 
tnx :+1:
